File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/java/clases_orientacion_objetos/clase_5_de_agosto_09/publico_privado.java
package ADSO2928707.portafolio_aprendices.keiner_lindarte_aro.instructor_henry.mis_programas.practicas.java.clases_orientacion_objetos.clase_5_de_agosto_09;

public class publico_privado {

    public static void main(String[] args) {
        
        Carro carro = new Carro("toyota","verde");
        System.out.println(carro.informacion());

    }
    
}

class Carro{
    String marca = "";
    String color = "";

    @SuppressWarnings("unused")
    private int precio = 1000000;

    Carro(String marca, String color){
        this.marca = marca;
        this.color = color;
    }

    String informacion(){
        return "El carro comprado es de la marca "+marca+", con el color "+color;
    }

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/python/clases_orientacion_objetos/5_clase_de_agosto_09/publico_privado.py
class Carro:
    marca = ""
    color = ""
    __precio = 1000000
    
    def __init__(self,marca,color):
        self.marca = marca
        self.color = color
        
    def informacion(self):
        print("El carro comprado es de la marca "+self.marca+", con el color "+self.color)

carro = Carro("toyota","verde")
carro.informacion()
    
    
        